src/SFA.DAS.EmployerIncentives/Application/Commands/CreateApplication/CreateApplicationHandler.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using SFA.DAS.EmployerIncentives.Application.Command.CreateApplication;
+using SFA.DAS.EmployerIncentives.Interfaces;
+using SFA.DAS.EmployerIncentives.Models.Commitments;
+using SFA.DAS.EmployerIncentives.Models.EmployerIncentives;
+
+namespace SFA.DAS.EmployerIncentives.Application.Commands.CreateApplication
+{
+    public class CreateApplicationHandler : IRequestHandler<CreateApplicationCommand, Guid>
```

Comment:
Doesnt appear to be any tests for this command handler

Reply:
Our original intent is to add the majority of the tests in the Acceptance Tests. this gives us much better coverage of the entire process. This means we can refactor fully without having to rewrite every layer in the application.

Saying this, we have had some issues with WireMock where it cannot easily test the body of a request. On some requests we have got round this by testing the string equivalent of the body. Therefore we are expecting to add unit tests when generating bodies